addons/auth_totp/data/mail_templates_data.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="mail_template_totp_invite" model="mail.template">
+        <field name="name">TOTP on User: Invite by email</field>
+        <field name="model_id" ref="base.model_res_users" />
+        <field name="subject">Invitation to activate two-factor authentication on your Odoo account</field>
+        <field name="partner_to">${object.partner_id.id}</field>
+        <field name="lang">${object.partner_id.lang}</field>
```

Comment:
Well they will have to convert it to QWeb. But mail templates still exist. They will use QWeb rendering instead of Jinja but overall behavior is the same.